README.md

```diff
@@ -112,6 +112,43 @@ title,content
 My Title,Hello World!
 ```
 
+### Throttling
+
+Maintenance Tasks often modify a lot of data and can be taxing on your database.
+The gem provides a throttling mechanism that can be used to throttle a Task when
+a given condition is met. Specify the throttle condition as a Proc:
+
+```ruby
+# app/tasks/maintenance/update_posts_throttled_task.rb
+module Maintenance
+  class UpdatePostsThrottledTask < MaintenanceTasks::Task
+    throttle_on -> { DatabaseStatus.unhealthy? }, backoff: 1.minute
```

Comment:
Do we really need a Proc? I feel like a block would be more Rubyish?

Reply:
Nope, Job Iteration says in their documentation that a Proc should be used so I went with that, but there's no reason we can't take a block for our API. I'll touch this up and you can let me know what you think.